start.php

```diff
@@ -170,7 +172,10 @@
 		fs_update_sdk_newest_version( $this_sdk_relative_path, $fs_active_plugins->plugins[ $this_sdk_relative_path ]->plugin_path );
 
 		$is_current_sdk_newest = true;
-	} else if ( version_compare( $fs_active_plugins->newest->version, $this_sdk_version, '<' ) ) {
+
+        // Try to reorder plugins so that the plugin containing the newest SDK version will be loaded first.
```

Comment:
@fajardoleo So it will mean we intend to redirect for the time, `$fs_active_plugins->newest` is not set. Event if this becomes the only Freemius SDK in the whole plugins/themes installs? If so, then I suggest to enrich the comment.